examples/amp-story/player-local-attachment.html

```diff
@@ -0,0 +1,55 @@
+<html>
```

Comment:
Do you think it'll be useful to check-in this sample?

Reply:
Check-in?

Reply:
Ah nvm, removed.